staging/src/k8s.io/client-go/discovery/cached/memory/memcache.go

```diff
@@ -190,16 +190,29 @@ func (d *memCacheClient) refreshLocked() error {
 		return err
 	}
 
+	wg := &sync.WaitGroup{}
+	resultLock := &sync.Mutex{}
```

Comment:
```
var resultLock sync.Mutex
```

works fine here, I think? No need to change it, we'll just wait for an "ineffassign" fix...